site/en/tutorials/images/transfer_learning.ipynb

```diff
@@ -1051,7 +1051,6 @@
         "predictions = model.predict_on_batch(image_batch).flatten()\n",
         "\n",
         "# Apply a sigmoid since our model returns logits\n",
```

Comment:
Maybe you should remove this comment since we're not applying a sigmoid anymore ...